integration/tracing/tracing-opentelemetry/src/main/java/org/apache/cxf/tracing/opentelemetry/AbstractOpenTelemetryClientProvider.java

```diff
@@ -141,7 +141,7 @@ protected void stopTraceSpan(final TraceScopeHolder<TraceScope> holder, final Th
 
             span.setStatus(StatusCode.ERROR);
             if (ex != null) {
-                span.recordException(ex, Attributes.of(ExceptionAttributes.EXCEPTION_ESCAPED, true));
+                span.recordException(ex);
```

Comment:
`ExceptionAttributes.EXCEPTION_ESCAPED` was removed in opentelemetry-semconv-incubating 1.30.0-alpha. Its docs said since longer that it was deprecated and "It’s no longer recommended to record exceptions that are handled and do not escape the scope of a span.", so just removing the attribute, given the hardcoded `true`, should be fine - see https://opentelemetry.io/docs/specs/semconv/registry/attributes/exception/

Reply:
Thanks for the details, @reta.

Does CXF 4.2 have any ETA?

> I think we could incorporate that into 4.1.x line as well but without bringing 1.32.0

Interesting idea. I will try next week whether it solves our problem.

Reply:
> Does CXF 4.2 have any ETA?

Yes, we hope to get it out of the door before end of the year (tentatively, for sure).

> Why do we rely on SNAPSHOT dependencies in a stable branch?

The branch is development mode, it is acceptable to have the SNAPSHOTs during the development cycle. We've just released `cxf-cxj-utils`, in preparation for release, should be all set now.